po/de.po

```diff
@@ -0,0 +1,250 @@
+# German translations for PACKAGE package
+# German messages for PACKAGE.
+# Copyright (C) 2015 THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
```

Comment:
@hzulla Do we need to fill out the actual information here? Also, you say this file is auto-generated; what is the process to update this if we need to add more text?